README.md

```diff
@@ -1,6 +1,21 @@
-# SonarLint IntelliJ Plugin
-
-[![Build Status](https://travis-ci.org/SonarSource/sonarlint-intellij.svg?branch=master)](https://travis-ci.org/SonarSource/sonarlint-intellij)
+SonarLint IntelliJ Plugin 
+[![Build Status](https://travis-ci.org/SonarSource/sonarlint-intellij.svg?branch=master)](https://travis-ci.org/SonarSource/sonarlint-intellij) [![Quality Gate](https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij&metric=alert_status
+)](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij)
+=========
+SonarLint is an IDE extension that helps you detect and fix quality issues as you write code.
```

Comment:
Any special reasoning for not using the full-form description from [Marketplace](https://plugins.jetbrains.com/plugin/7973-sonarlint) ? (I would have, maybe naively, thought the richer the better)

Reply:
My thoughts were:
- This repository may be primarily visited by Devs interested in the SLI codebase. "Build" and "Develop" content sections are highly relevant to them.
- It may also be visited by devs discovering Sonarlint and they need an intro. 
- We want to address both audiences
Decided to go for a quick description complemented with a useful links section:
- easier to digest than a long text paragraph
- prevent dilution of existing text content (find the right balance for both targets).
- quick description should entice devs who discover SL to click one of the useful links

Reply:
Ok! As discussed let's simply add the second part of the value prop, and agreed that's it's enough as is.